puphpet/config.yaml.template

```diff
@@ -60,10 +71,20 @@ vagrantfile-local:
         host: detect
 server:
     install: '1'
+    apt-ppa-repos:
+        - ppa:git-core/ppa   # Provides newer git-core. -bp
+        - ppa:chris-lea/node.js  # Provides newer node and npm. -bp
     packages:
-        - vim
-        - memcached
         - curl
+        - emacs
```

Comment:
Individual projects can just flat out delete the lines that will never apply to them, there's no need for the comments if the skeleton is always around for reference (with this list, that is). 

I again also don't want devs to have to opt IN TO much at all, preferably nothing. The skeleton is meant to be over robust. You'll certainly get more from it than any given project needs, but again it's easier to just wipe things out than figure out which related bits to pull in. 






> On Jan 31, 2015, at 10:22 AM, Andrew Lechowicz <notifications@github.com> wrote:
> 
> In puphpet/config.yaml.template:
> 
> >          - curl
> > +        - emacs
> Sorry about that, the comments didn't show in the Github diff. Should we consider commenting out packages that we know aren't needed in every project then? It would be much easier to hunt for them then and we're not waisting time each time a dev builds the box installing packages no one will ever use .
> 
> —
> Reply to this email directly or view it on GitHub.
>